include/arpa/inet.h

```diff
@@ -50,6 +50,7 @@ in_addr_t   inet_addr(FAR const char *cp);
 in_addr_t   inet_network(FAR const char *cp);
 
 FAR char   *inet_ntoa(struct in_addr in);
+FAR char   *inet_ntoa_r(struct in_addr in, FAR char *buf);
```

Comment:
most of the variants that I lookup in the Internet give me `char * inet_ntoa_r(struct in_addr in, char *buf, socklen_t size);`, so why `size` is not added in this variant?

Reply:
Since `inet_ntoa_r` isn't really specified in POSIX I tried to make it close as the GNU `ether_ntoa_r` variant.

Reply:
Yeah, but
http://www.qnx.com/developers/docs/7.0.0/index.html#com.qnx.doc.neutrino.lib_ref/topic/i/inet_ntoa_r.html
https://www.ibm.com/docs/en/i/7.3?topic=ssw_ibm_i_73/apis/tsinntoa.htm
https://nxmnpg.lemoda.net/3/inet_ntoa_r
https://github.com/espressif/esp-idf/blob/master/examples/protocols/sockets/tcp_server/main/tcp_server.c
https://asf.microchip.com/docs/latest/sam4s/html/inet_8h.html
https://www.unix.com/man-page/freebsd/3/inet_ntoa_r/

I would stick to some more common implementation even if it not closer to GNU `ether_ntoa_r` variant.

Reply:
Ah I see, still many small variants though.
Do you agree on the following?
`char * inet_ntoa_r(struct in_addr in, FAR char *buf, size_t size);`
I prefer to use `size_t` just like `strncpy`

Reply:
Yes. That should be fine!